src/create-menu/createMenu.js

```diff
@@ -194,30 +201,122 @@ function getWppListMenu(props) {
+                    menuText += `\n\n${channelBoldTags.open}${option}${channelBoldTags.close}. ${menuOptions[i]}`;
+                } else {
+                    menuText += `\n${channelBoldTags.open}${option}${channelBoldTags.close}. ${menuOptions[i]}`;
+                }
             }
         }
     }
-    let textMenu = { text: menuText };
-    return textMenu;
+    return menuText;
+}
+
+function buildMenuTextOptionsWhenIsObject(props, config) {
+    let { channelBoldTags, menuFields } = props;
+    let menuOptions = menuFields.options;
+    let menuText = '';
+    try {
+        const sections = Object.keys(menuOptions);
+        let totalItens = getNumberOfOptions(menuOptions);
+        if (config.orderOptions === 'desc') {
+            start = totalItens - 1;
+            let i = start;
+            for (let k = 0; k < sections.length; k++) {
+                menuText += sections[k] ? `\n${sections[k]}\n` : '\n';
+                for (let j = 0; j < menuOptions[sections[k]].length; i--, j++) {
+                    if (menuOptions[sections[k]][j][0] === '\n') {
+                        menuOptions[sections[k]][j] = menuOptions[sections[k]][
+                            j
+                        ].replace('\n', '');
+                        menuText += `\n\n${channelBoldTags.open}${i + 1}${
+                            channelBoldTags.close
+                        }. ${menuOptions[sections[k]][j]}`;
+                    } else {
+                        menuText += `\n${channelBoldTags.open}${i + 1}${
+                            channelBoldTags.close
+                        }. ${menuOptions[sections[k]][j]}`;
+                    }
+                }
+                menuText += k < sections.length - 1 ? '\n' : '';
+            }
+        } else {
+            let i = 1;
+            for (let k = 0; k < sections.length; k++) {
+                menuText += sections[k] ? `\n${sections[k]}\n` : '\n';
+                for (let j = 0; j < menuOptions[sections[k]].length; j++, i++) {
+                    let option = i;
+                    if (menuFields.isSurvey) {
+                        option = totalItens - i;
+                    }
+                    if (menuOptions[sections[k]][j][0] === '\n') {
+                        menuOptions[sections[k]][j] = menuOptions[sections[k]][
+                            j
+                        ].replace('\n', '');
+                        menuText += `\n\n${channelBoldTags.open}${option}${
+                            channelBoldTags.close
+                        }. ${menuOptions[sections[k]][j]}`;
+                    } else {
+                        menuText += `\n${channelBoldTags.open}${option}${
+                            channelBoldTags.close
+                        }. ${menuOptions[sections[k]][j]}`;
+                    }
+                }
+                menuText += k < sections.length - 1 ? '\n' : '';
+            }
+        }
+    } catch (error) {
```

Comment:
Here you can use a finally clause instead of a catch block:

```js
let menuText = "";
try {
    // try to do your logic with menuText
} finally {
    return menuText;
}
```